selfdrive/loggerd/logger.h

```diff
@@ -31,6 +31,7 @@ typedef struct LoggerHandle {
 
 typedef struct LoggerState {
   pthread_mutex_t lock;
+  kj::ArrayPtr<capnp::byte> init_data;
```

Comment:
dangling pointer.
I have submitted a PR  https://github.com/commaai/openpilot/pull/19942 and can use that as a cache